packages/stream-api/src/streamApi.ts

```diff
@@ -32,9 +32,15 @@ export class StreamApi {
   private onRequestDispatcher = new EventDispatcher<RequestMessage>();
   private messageSubscription?: Disposable;
 
+  // Tracks a period of time with no interaction (requests or responses)
+  // indicating that we have lost connection, and the websocket client
+  // used will no longer function as expected
+  private uninteractiveTimeout?: number;
+
   public constructor(
     private websocket: WebSocketClient = new WebSocketClientImpl(),
-    private loggingEnabled = false
+    private loggingEnabled = false,
+    private uninteractiveThreshold: number = 60 * 1000
```

Comment:
Thinking it might make sense to have this value specified by FSS on `StartStream` messages if we'd want to move forward with this approach.